packages/docusaurus/src/client/exports/useEventBus.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import {useEffect} from 'react';
```

Comment:
Do this have to be in core namespace ? :(

It's too generic to be included as part of `@docusaurus`. Maybe you can publish as npm package :v
I'd rather have it as part of theme-classic. Because our goal is to make core as small as possible and not everyone gonna use this.

Reply:
Of course, this is all the more just a preliminary version, I had no idea how this solution (event bus) would suit you!

> Maybe you can publish as npm package :v

Can it just move to the theme level? (By analogy with a TOC hook)